tests/lib/rules/no-unused-prop-types.js

```diff
@@ -1387,15 +1319,16 @@ ruleTester.run('no-unused-prop-types', rule, {
       code: [
         'const PagingBlock = function(props) {',
         '  return (',
-        '    <SomeChild {...props} />',
+        '    <span>{...props}</span>',
         ' );',
         '};',
 
         'PagingBlock.propTypes = {',
         '  nextPage: PropTypes.func.isRequired,',
         '  previousPage: PropTypes.func.isRequired,',
         '};'
-      ].join('\n')
+      ].join('\n'),
+      parser: 'babel-eslint'
```

Comment:
jsx spread works in ES6 or above in the default parser with `parserOptions: { jsx: true }` - there's definitely no need for `babel-eslint` for it.